x-pack/plugins/security_solution/public/timelines/components/timeline/search_or_filter/pick_events.tsx

```diff
@@ -139,17 +140,15 @@ const PickEventTypeComponents: React.FC<PickEventTypeProps> = ({
 
   const indexesPatternOptions = useMemo(
     () =>
-      [
-        ...configIndexPatterns,
-        ...kibanaIndexPatterns.map((kip) => kip.title),
-        signalIndexName,
-      ].reduce<Array<EuiComboBoxOptionOption<string>>>((acc, index) => {
+      [...kibanaIndexPatterns.map((kip) => kip.title), signalIndexName].reduce<
+        Array<EuiComboBoxOptionOption<string>>
+      >((acc, index) => {
         if (index != null && !acc.some((o) => o.label.includes(index))) {
```

Comment:
I know you didn't write this, but isn't it always `label === index`?

Reply:
its saying `acc` shouldn't already have a `label` that is equal to that `index`. makes sense to me but we can run it together if you'd like and poke at it 😄